content/en/docs/sdk-utilities/s3.md

```diff
@@ -0,0 +1,430 @@
+---
+title: "Amazon S3 Utilities"
+linkTitle: "Amazon S3"
+---
+
+# Transfer Managers
+
+The {{% alias service="S3long" %}} upload and download managers can break up large objects, so they can be transferred
+in multiple parts, in parallel. This makes it easy to resume interrupted transfers.
+
+## Upload Manager
+
+The {{% alias service="S3long" %}} upload manager determines if a file can be split into
+smaller parts and uploaded in parallel. You can customize the number of parallel uploads and the size of the uploaded
+parts.
+
+The following example uses the {{% alias service="S3" %}} `Uploader` to upload a file.
+Using `Uploader` is similar to the `s3.PutObject()` operation.
+
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+import "github.com/aws/aws-sdk-go-v2/service/s3"
+import "github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig()
+if err != nil {
+	log.Printf("error: %v", err)
+	return
+}
+
+client := s3.NewFromConfig(cfg)
+
+uploader := manager.NewUploader(client)
+result, err := uploader.Upload(context.Background(), &s3.PutObjectInput{
+	Bucket: aws.String("my-bucket"),
+	Key:    aws.String("my-object-okey"),
+	Body:   uploadFile,
+})
+```
+
+### Configuration Options
+
+When you instantiate an `Uploader` instance using [NewUploader]({{% apiref "feature/s3/manager#NewUploader" %}}), you
+can specify several configuration options to customize how objects are uploaded. Options are overridden by providing one
+or more functional arguments to `NewUploader`. These options include:
+
+* `PartSize` &ndash; Specifies the buffer size, in bytes, of each part to upload. The minimum size per part is 5 MB.
+* `Concurrency` &ndash; Specifies the number of parts to upload in parallel.
+* `LeavePartsOnError` &ndash; Indicates whether to leave successfully uploaded parts in {{% alias service="S3" %}}.
+  
+The `Concurrency` value limits the concurrent number of part uploads that can occur for a given `Upload` call. 
+This is not a global client concurrency limit. Tweak the `PartSize` and `Concurrency` configuration values to find the
+optimal configuration. For example, systems with high-bandwidth connections can send bigger parts and more uploads in parallel.
+
+For example, your application configures `Uploader`  with a `Concurrency` of setting of `5`. Your application then 
+calls `Upload` from two different goroutines, the result will be `10` concurrent part uploads
+(2 gorutines * 5 `Concurrency`).
+
+{{% pageinfo color="warning" %}}
+Your application is expected to limit the concurrent calls to `Upload` to prevent application resource exhaustion.
```

Comment:
A little more detail on what a sensible limit would be, or how to determine one, would be welcome.